app/models/user.js

```diff
@@ -14,6 +13,7 @@ var Promise = require('bluebird')
   , mkKey = require("../support/models").mkKey
   , _ = require('underscore')
   , validator = require('validator')
+  , bcrypt = require('bcrypt-then')
```

Comment:
Please replace bcrypt-then library to a native brcypt and promisify it `bcrypt = Promise.promisifyAll(require(‘<bcrypt-library>'))`. I'm breaking a style of micro-modules if we can replace that library with one or two lines of our code.